social_core/backends/okta.py

```diff
@@ -0,0 +1,79 @@
+"""
+Okta OAuth2 and OpenIdConnect:
+    https://python-social-auth.readthedocs.io/en/latest/backends/okta.html
+"""
+import sys
+if sys.version_info[0] < 3:
+    import urlparse as urlparse
+else:
+    import urllib.parse as urlparse
+
+import requests
+from six.moves.urllib.parse import urljoin
+from jose import jwk, jwt, jws
+from jose.jwt import JWTError, JWTClaimsError, ExpiredSignatureError
+from jose.exceptions import JWSSignatureError
+
+from ..utils import append_slash
+from .oauth import BaseOAuth2
+from .open_id_connect import OpenIdConnectAuth
```

Comment:
importing the OpenIdConnectAuth may fail if the extra option "openidconnect" hasn't been installed (https://python-social-auth.readthedocs.io/en/latest/installing.html#using-the-extras-options).
Could the import be in a try:except: block so that if it fails, the class `OktaOpenIdConnect` is not defined ?

Reply:
I implemented the OpenIdConnect into its own file, like its done with the Google OpenIdConnect.